### DISCUSSION TITLE
How to negate a schedule

### DISCUSSION THREAD

**Discussion by mbrevda** [renovatebot/renovate#17769] [Request Help]
Hi - is it possible to negate a schedule? Use case: run every Monday, unless one of the following dates (national holidays)

**Comment by rarkins** [comment]
Not that I'm aware of. We also support cron format schedules, but it also doesn't have a "except on X date" provision

**Reply by mbrevda** [reply]
Thanks! Is there any option for custom code or perhaps checking with an external server before running?

**Reply by rarkins** [reply]
Not that I can think of

**Reply by mbrevda** [reply]
Would it make sense to add something? Say a script that can be executed and it's exit status would determine if renovate should run?

**Reply by rarkins** [reply]
My initial instinct is no for that. What about the option for excluded schedules instead?

**Reply by mbrevda** [reply]
I was having trouble getting Later to understand what I wanted...

**Reply by rarkins** [reply]
I mean adding new field like exceptSchedle which can include regular Later or cron syntax

**Reply by usmonster** [reply]
My team and I would be interested in this, and maybe some extensions or included defaults with lists of national holiday dates? At what point shall we make it a feature request? Thanks!
